common/tools/eslint-plugin-azure-sdk/src/configs/index.ts

```diff
@@ -28,6 +28,7 @@ export = {
       "@azure/azure-sdk/ts-apisurface-supportcancellation": "error",
       "@azure/azure-sdk/ts-config-include": "error",
       "@azure/azure-sdk/ts-doc-internal": "error",
+      "@azure/azure-sdk/ts-doc-internal-private-member": "error",
```

Comment:
one suggestion for this rule is to make it "warn", as the violations are not too concerning, but nice to know about.  Could you change this to "warn" please?

Reply:
I agree with making this a warning-level lint rather than an error. At the end of the day what we want to warn the developer of is that the `@internal` tag doesn't do anything on private members. If it did something _bad_, then we'd make it an error and make a point of cleaning up the codebase.